content/news/XXXX-XX-XX-github-issues-migration.md

```diff
@@ -0,0 +1,85 @@
+title: Migrated to Github issues
+authors: Nikolaus Einhauser
+tags: infrastructure, development, contribute
+comments: yes
+status: draft
+
+
+Yes everybody, we're officially moving our bug tracking away from Launchpad to
```

Comment:
This is using present progressive, but you use past tense elsewhere. We should use one form consistently.

Reply:
I agree. I'll fix it. past tense makes more sense considering we'll publish this after the migration has been completed.